linux/guest_customization/gosc_support_matrix.yml

```diff
@@ -17,9 +19,18 @@ Rocky:
       7.0.1: N/A
       7.0.2: N/A
       7.0.3: 18700403
+  '9.0':
+    vcenter:
+      6.7.0: 19832280
```

Comment:
From https://docs.vmware.com/en/VMware-vSphere/6.7/rn/vsphere-vcenter-server-67u3r-release-notes.html
the build number is 19832974

Reply:
This is the VPXD build number, not vCenter build number. Please check here: https://kb.vmware.com/s/article/2143838.